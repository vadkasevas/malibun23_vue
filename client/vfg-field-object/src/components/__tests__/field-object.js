import Vue from "vue";
import FieldObject from "../field-object";

describe("field-object.vue", () => {
  const Constructor = Vue.extend(FieldObject);
  const vm = new Constructor({
    propsData: {
      value: {},
      schema: {},
      model: {
        field1: "value1"
      },
      disabled: false,
      options: {}
    }
  }).$mount();
  test("should have 1 table", () => {
    expect(vm.$el.querySelectorAll("table").length).toEqual(1);
  });
  test("should match the snapshot", () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
