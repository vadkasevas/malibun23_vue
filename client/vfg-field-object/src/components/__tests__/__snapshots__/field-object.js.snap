// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`field-object.vue should match the snapshot 1`] = `
<div>
  <div>
    <table
      id=""
    />
     
    <select>
      <option
        value="string"
      >
        String
      </option>
       
      <option
        value="number"
      >
        Number
      </option>
       
      <option
        value="boolean"
      >
        Boolean
      </option>
    </select>
     
    <input
      type="text"
    />
     
    <input
      type="button"
      value="Add key"
    />
  </div>
</div>
`;
